manifests/server.pp

```diff
@@ -51,11 +55,13 @@
     -> Class['hyperglass::server::install']
   }
 
+  include hyperglass::hyperglassdir
```

Comment:
You could use `require` here instead and remove the relationship below.

Reply:
thanks, fixed it